src/main/java/com/keshavarzi/airplanestats/model/RouteEntity.java

```diff
@@ -0,0 +1,63 @@
+package com.keshavarzi.airplanestats.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "route", schema = "user_data")
+public class RouteEntity {
+
+    @Id
+    @Nonnull
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", table = "route", unique = true, nullable = false)
+    private Long routeId;
+
+    // Returning a reference to a mutable object value stored in one of the object's fields exposes the internal representation of the object.
+    // If instances are accessed by untrusted code, and unchecked changes to the mutable object would compromise security or other important properties,
+    @Nonnull
+    @Getter(AccessLevel.NONE)
+    @Setter(AccessLevel.NONE)
+    @ManyToOne(targetEntity = UserEntity.class, cascade = CascadeType.ALL)
+    @JoinColumn(name = "user_id", referencedColumnName = "id", table = "route", nullable = false, unique = true,
+            foreignKey = @ForeignKey(name = "route_user_id_fk"))
+    private UserEntity userEntity;
```

Comment:
Why is a `UserEntity` a part of a Airplane route?

Reply:
It's so that a user can have a history of all the trips.  Is there something else you had in mind?